src/main/java/org/mondemand/Client.java

```diff
@@ -383,21 +383,28 @@ public void addTransportsFromDefaultConfigFile()
    * adds transports from a configuration file. the format of the file is:
    * MONDEMAND_ADDR="ip_addr_1, ip_addr_2, ..."
    * MONDEMAND_PORT="port"
+   * (optional) MONDEMAND_TTL="ttl"
    *
    * @param configFileName - configuration file name.
    * @throws Exception if file does not exist, or if there is a problem reading
    *        the file, or either port or address is missing, or if port cannot
    *        be converted to number, or if addresses cannot be converted to valid
-   *        hosts, or if a transport cannot be created for addresses/port
+   *        hosts, or if ttl cannot be converted to number in valid range,
+   *        or if a transport cannot be created for addresses/port
    *        specified in the file.
    */
   public void addTransportsFromConfigFile(String configFileName)
```

Comment:
That's how the erlang library does it.  Although it does it in two steps, one to load the config a second to restart all the "transports".  That way it can check if the config is bad and only restart if it's good (although I mostly do that by hand via a shell).  Either way, I think this PR can be merged and maybe a new one with this additional functionality?